### PR TITLE
Login registration

### PR DESCRIPTION
**Pull request by IJumpAround**
## Login Page
* I think ideally this will be rendered over the current page when the login button is clicked instead of redirecting.
* Some functionality skeleton in place such as click event handlers.

## Registration Page
* Still needs more fields I think, but this is good for a demo.
* Fields: First name, last name, password, DOB, email.
* Nice warm welcome message

#### Dependencies
* React-bootstrap: Added for an easier time making the modal stuff. From what I've read it can be used without interfering with anything we are using.
* FontAwesome: Fonts/Images pack. When imported one component at a time there is minimal cost.

### CODE REVIEW

File: reactClient/src/Components/homePage/HomePage.js [hunk 1]
```diff
@@ -6,15 +6,13 @@ import '../homePage/HomePage.css';
 
   render() {
     return (
-      
-     <body>
```
Comment (IJumpAround) [inline reactClient/src/Components/homePage/HomePage.js:10]: This was throwing errors in the console, so I removed the `<body>` tag.

File: reactClient/src/Components/searchComponents/DynamicCards.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ import {
 									/>
 									<CardContent>
 										<Typography  gutterBottom>
-										<img src={require('../../img/stockdormimage.jpg')} width="80%"  alt/>
+										<img src={require('../../img/stockdormimage.jpg')} width="80%"  alt=""/>
 										</Typography>
```
Comment (IJumpAround) [inline reactClient/src/Components/searchComponents/DynamicCards.js:47]: This was also throwing errors about assigning a boolean value to alt. So I set it to an empty string.

**Comment by sollazzo08** [comment]
Looking good!

**Review by sollazzo08 — APPROVED** [review]
Looks good!

**Pull request merged**